Accesibility

Timowest12 opened the pull request:
Made changes 
- Background color for contrast
- Button color for contrast
- Alt = "alt text"
- Multmedia
- Headings tags
- Animation
- Text headings
- Moving content
- Interaction resize
- Page title

Mihndim2020 reviewed (changes requested):
### Hi @Timowest12, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Linters are passing
:heavy_check_mark: The project is running.

There are some issues that you still need to work on to get this project approved, but you are almost there!
![ALMOST-MINION](https://user-images.githubusercontent.com/65852621/133680327-431cb21d-ad7c-483a-84eb-21b733980a04.png)



# Status: Requires Changes ♻️
_Check the comments below._

- [ ] Please you have to prepare a report listing the points you have checked, and the results of your evaluation for each point. Your report should include;
  - Page titles,
  - Image text alternatives** (<img alt="alt text">),
  - Text headings,
  - Color contrast,
  - Resize,
  - Interaction,
  - Moving content,
  - Multimedia,
  - The basic structure of the page. As listed in the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md). 
  *****For each of this points***** you need to include the results of your evaluation:
   - If everything was ok, make a statement about that (e.g. "Page titles - no issues found").
   - If you needed to change something, [link to the corresponding line of code in your pull request comments.](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/)
   - [ ] Add your report to the description of your pull request. In  order words, each of those points above, make sure your site is checked against it, if it is ok, in your pull request, you make a comment "No issues found" against that point, if there is an issue, make sure you update(fix the issue in) your code and in the pull request, link to the line(s) of code where you have done the update(fix). Your report shows that you have carried out only one check which is color contrast, please kindly implement the requested changes and resubmit for re-review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Timowest12 commented:
Made changes

-Background colour for contrast:

 I used a site called https://color.a11y.com/Contrast/ to check for contrast issues on my site, I found that the background colour created a lousy contrast with other colours. So i changed it to suit the difference better.

-Button colour for contrast

Similar to the background colour, the background colour of my buttons was also causing accessibility issues, so I changed the colour the suit the contrast better.

-Alt = "alt text"

I did not use any IMG tags on my site yet, so alt's are unnecessary.

-Multimedia

No multimedia, so no issues found.

-Headings tags

I noticed I used two h1 tags on my page. So I changed it to 1.

-Animation

I deleted an animation that might have annoyed my visitors.

-Text headings

no issues found

-Moving content

no issues found

-Interaction resize

no issues found

-Page title

no issues found

SajjadAhmad14 reviewed (approved):
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Timo Wester, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
